docs/modules/ROOT/pages/how-to/configure-alertmanager.adoc

```diff
@@ -0,0 +1,445 @@
+    timeIntervals:
+    - times:
+      - startTime: "00:00" <7>
+        endTime: "09:00"
+      - startTime: "17:00"
+        endTime: "24:00"
+      weekdays:
+      - "monday:friday"
+    - weekdays:
+      - "saturday"
+      - "sunday"
+----
+<1> The default receiver.
+All alerts that don't match any team will be routed to this receiver.
+<2> Don't further evaluate the remaining routes.
+The alert will be routed to the given team and won't appear in the default "catch-all" receiver.
+`false` is the default value and only set for demonstration purposes.
+<3> Matches alerts with the label `team=database`, given from the parent route, and `severity=critical`.
+<4> Sets this route as an additional receiver for the alert.
+The alert will be routed to the `database-team-pagerduty` receiver and the parent Slack receiver.
+<5> Mutes the Slack alerts for the back-end team during non-office hours.
+When a route is muted it won't send any notifications, but otherwise act normally, this includes ending the route-matching process if the `continue` option isn't set.
+<6> Defines a time interval that can be referenced in the `muteTimeIntervals` field of a route.
+<7> All times are in UTC, no other time zones are supported.
+
+== Test Alerting [[test-alerting]]
+
+All changes to alerting routes and receivers should be tested to ensure that receivers are able to correctly receive alerts.
+
+. Set the namespace you created the `AlertmanagerConfig` in.
++
+[source,bash]
+----
+APP_NAMESPACE=my-namespace
+----
+
+. Create a test alert that's always firing.
++
+[source,bash]
+----
+kubectl -n "${APP_NAMESPACE}" apply -f - <<YAML
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: test-alert
+spec:
+  groups:
+  - name: test-alert
+    rules:
+    - alert: TestAlert
+      expr: vector(1)
+      for: 10s
+      labels: {} <1>
+      annotations:
+        summary: "Test Alert"
+        description: "This is a test alert"
+YAML
+----
+<1> Add labels to the alert to route it to the correct receiver.
+Usually the default receiver is used if no labels are set.
+
+. Check your configured receivers to see if the alert was received.
++
+See <<troubleshooting>> for debugging tips.
+
+. Delete the test alert.
++
+[source,bash]
+----
+kubectl -n "${APP_NAMESPACE}" delete prometheusrule test-alert
+----
+
+. Repeat the test for all receivers by adding the appropriate labels to the alert.
+
+== Troubleshooting [[troubleshooting]]
+
+OpenShift currently doesn't expose any logs or status information for the Alertmanager.
+This makes it difficult to debug issues with Alertmanager.
+The following section describes some common issues and how to debug them.
+
+. Check all secret references.
++
+One missing secret reference blocks the creation of all the receivers in the `AlertmanagerConfig`.
+
+. Check the secret values.
++
+Wrong login credentials or missing certificates can cause the Alertmanager to fail to send alerts.
```

Comment:
Will add a debug your webhook section in a separate PR.

Reply:
Added in https://github.com/appuio/appuio-cloud-docs/pull/141/commits/7618e3135fe88b508320d44955d82f88daf5f19b